src/widget-sources/types.ts

```diff
@@ -131,7 +132,8 @@ export interface TimeSeriesRequestOptions extends BaseRequestOptions {
   column: string;
   stepSize: GroupDateType;
   stepMultiplier?: number;
-  operation?: 'count' | 'avg' | 'min' | 'max' | 'sum';
+  operation?: 'count' | 'avg' | 'min' | 'max' | 'sum' | 'custom';
```

Comment:
See above.

Reply:
I've just realized that `operation` is mandatory, at least for category/timeseries/formula`

Can we remove `?` ... i think we should, just asking for opinions

Reply:
@zbigg for tileset widgets it is optional and defaults to 'count' ...

https://github.com/CartoDB/carto-api-client/blob/fd34134af9dfc48b88dae6f4e0ea26f484b273cf/src/widget-sources/widget-tileset-source-impl.ts#L121-L123

... I'm not sure if changing that would break things, or whether there are legacy reasons vs. just an unplanned difference. So it might be safer to default to 'count' everywhere. I'm OK with either but would probably want to make this a v0.5 -> v0.6 bump to be safe if we're making the property required for tilesets.